esmvaltool/cmorizers/obs/cmorize_obs_wfde5.py

```diff
@@ -0,0 +1,135 @@
+"""ESMValTool CMORizer for WFDE5 data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://doi.org/10.24381/cds.20d54e34
+
+Last access
+    20210416
+
+Download and processing instructions
+    Download the following variables from the cds:
+        Near-surface air temperature ("CRU")
+        Rainfall flux ("CRU" as well as "CRU and GPCC")
+        Snowfall flux ("CRU" as well as "CRU and GPCC")
+    unzip the downloaded files
+    rename to follow syntax '{raw_name}_WFDE5_{reference}_*_v1.0.nc'
+"""
+
+import copy
+import logging
+import re
+from pathlib import Path
+
+import iris
+from cf_units import Unit
+from esmvalcore.preprocessor import daily_statistics, monthly_statistics
+
```

Comment:
When I pulled the branch to test it yesterday, this line was only showing up as 
`from esmvalcore.preprocessor import monthly_statistics`

That made the cmorizer crash, of course. I added the `daily_statistics` manually, and then it ran fine. 
This is weird. @valeriupredoi, could you check what you get when you pull the branch?

Reply:
it's possible you pulled a local copy, or that your git cache may be a bit special :grin: Anyway I pulled the branch from remote and that was there all fine, I now merged `master` in so all should be tiptop. I'll review it now :+1: